src/components/Footer/FooterBase.tsx

```diff
@@ -0,0 +1,130 @@
+import Link from '../Link';
+import logoTagline from '../../stories/assets/logo-bsc-tagline.svg';
+import { RequireOnlyOne, NavItem } from '../../types';
+import Facebook from '../../stories/assets/icon-facebook.svg';
+import Twitter from '../../stories/assets/icon-twitter.svg';
+import LinkedIn from '../../stories/assets/icon-linkedin.svg';
+import YouTube from '../../stories/assets/icon-youtube.svg';
+import NewWindowIcon from '../../../public/assets/images/icon-new-window.svg';
+
+interface LegalLinksBase extends NavItem {
+  children?: LegalLinkItem[];
+}
+
+export type LegalLinkItem = RequireOnlyOne<LegalLinksBase, 'to' | 'href'>;
+
+export interface FooterBaseProps {
+  legalLinkItems?: LegalLinkItem[];
+  legalLinkAriaLabel?: string;
+  corporateLink?: boolean;
+  customizeCookiesLink?: string;
+  complianceCode?: string;
+  socialMedia?: {
+    facebook?: string;
+    twitter?: string;
+    linkedIn?: string;
+    youTube?: string;
+  };
+  texts?: {
+    corporateLinkText?: string;
+    tagline?: string;
+    customCookiesLinkText?: string;
+    complianceCodeLabel?: string;
+    copyrightText?: string;
+  };
+}
+
+const FooterBase = ({
+  legalLinkItems,
+  legalLinkAriaLabel,
+  corporateLink,
+  texts,
+  customizeCookiesLink,
+  complianceCode,
+  socialMedia
+}: FooterBaseProps) => {
+  const socials = [
+    {
+      platform: 'Facebook',
+      socialLink: socialMedia?.facebook,
+      icon: Facebook,
+      defaultLink: 'https://www.facebook.com/BostonScientific'
+    },
+    {
+      platform: 'Twitter',
+      socialLink: socialMedia?.twitter,
+      icon: Twitter,
+      defaultLink: 'https://twitter.com/bostonsci'
+    },
+    {
+      platform: 'LinkedIn',
+      socialLink: socialMedia?.linkedIn,
+      icon: LinkedIn,
+      defaultLink: 'https://www.linkedin.com/company/boston-scientific'
+    },
+    {
+      platform: 'YouTube',
+      socialLink: socialMedia?.youTube,
+      icon: YouTube,
+      defaultLink: 'https://www.youtube.com/channel/UC6mvgpoLzpXU-tlbATlrXUw'
+    }
+  ];
+  return (
+    <div className="bsds-footer-base">
+      {!!legalLinkItems && (
+        <nav className="bsds-nav-footer-legal" aria-label={legalLinkAriaLabel}>
```

Comment:
let's ask catie if there's a name for this section that we could use as a default aria label value. the current naming doesn't stand out as being applied to a nav element to me.